sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusMessageTest.java

```diff
@@ -48,7 +48,7 @@ void messagePropertiesShouldNotBeNull() {
         // Assert
         Assertions.assertNotNull(serviceBusMessageData.getBody());
         Assertions.assertNotNull(serviceBusMessageData.getContext());
-        Assertions.assertNotNull(serviceBusMessageData.getProperties());
+        Assertions.assertNotNull(serviceBusMessageData.getApplicationProperties());
```

Comment:
do you have a test for that copy constructor? I have a feeling if you have a SBRMessage and start modifying annotated properties (ie. Subject), the original one will be modifed. (which isn't what you want).

Reply:
I have written test for it now.

Reply:
In the test you should modify and expect the values are different.

Reply:
AmqpAnnotatedMessage: Since this copy constructor will copy values, We have test in `AmqpAnnotatedMessageTest` which check for values.

But I have added additional test in `ServiceBusMessageTest` which will user setters on `ServiceBusMessage` to change the properties and do assersions.
I hope both will cover all the cases.